jc3mpserverconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "announce": {{announce}},
+  "description": "{{description}}",
+  "host": "0.0.0.0",
```

Comment:
This should have a matching setting under the jc3mpconfig.json that ties to the $ApplicationIPBinding.

Reply:
Are you confident that $ApplicationIPBinding can be changed through the manifest? When I tried it some time ago it was not possible

Reply:
I'm not 100% but thought it could be a hidden setting. Not editable though.

Reply:
Sure, hidden would work but then is there much point? The user would have to know how to edit GenericModule.kvp. Much better would be for Mike to surface this var in the manifest

Reply:
Alternatively in the meantime as I have done in other cases is have a manifest setting but not link it to $ApplicationIPBinding. Causes a visual mismatch with the endpoint reported in AMP but at least the user can change the actual binding in the UI

Reply:
I would think of they were concerned about changing the setting they'd know how to create it properly in the first place, no? Would be nice to have a setting you can edit. But that's like saying there's no point adding the ports because the user can't change them in the instance. :p

Reply:
Added as suggested, and it does show in the config.json when generated.  It is marked as hidden.